homeassistant/components/otbr/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "after_dependencies": ["hassio"],
```

Comment:
How did this file get alphabetically sorted keys?

Reply:
Fixed